tests/compile/distributed/test_sequence_parallelism.py

```diff
@@ -153,7 +153,10 @@ def ops_in_model_before(self):
         ]
 
     def ops_in_model(self):
-        if self.vllm_config.compilation_config.pass_config.enable_fusion:
+        if (
+            self.vllm_config.compilation_config.pass_config.fuse_norm_quant
+            and self.vllm_config.compilation_config.pass_config.fuse_act_quant
+        ):
```

Comment:
Just rmsnorm-quant here:
```suggestion
        if self.vllm_config.compilation_config.pass_config.fuse_norm_quant:
```